src/main/java/nerdhub/textilelib/mixins/MixinWorld.java

```diff
@@ -1,52 +1,18 @@
 package nerdhub.textilelib.mixins;
 
 import nerdhub.textilelib.eventhandlers.EventRegistry;
-import nerdhub.textilelib.events.entity.EntitySpawnedEvent;
 import nerdhub.textilelib.events.tick.WorldTickEvent;
-import net.minecraft.entity.Entity;
-import net.minecraft.entity.player.PlayerEntity;
-import net.minecraft.util.math.MathHelper;
 import net.minecraft.world.World;
-import org.spongepowered.asm.mixin.*;
-import org.spongepowered.asm.mixin.injection.*;
-import org.spongepowered.asm.mixin.injection.callback.*;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 import java.util.function.BooleanSupplier;
-import java.util.stream.Stream;
 
 @Mixin(World.class)
 public abstract class MixinWorld {
 
-    @Inject(method = "spawnEntity", at = @At("HEAD"), cancellable = true)
-    private void spawnEntity(Entity entity_1, CallbackInfoReturnable<Boolean> cir) {
```

Comment:
Won't this remove the ability for client side triggering of spawn event when in multi-player.

Reply:
it simply doesn#t exist anymore on the client, that's why I removed it.

Reply:
Well that is disappointing. Guess if we want to keep (or re-add) this event functionality we will have to do a packet parsing or see how else an entity is added client side.